data/faq.html

```diff
@@ -2626,7 +2864,7 @@ <h3 lang="tr">Sesli yönlendirmeyi duyamıyorum</h3>
           Arabic, Chinese (Traditional and Simplified), Czech, Danish, Dutch,
           Finnish, French, German, Greek, Hindi, Hungarian, Indonesian, Italian,
           Japanese, Korean, Polish, Portuguese, Romanian, Russian, Slovak,
-          Spanish, Swedish, Thai, Turkish.
+          Spanish, Swedish, Thai, Ukrainian, Turkish.
```

Comment:
Should 26 above be fixed? Is there an official list somewhere? Did you count it yourself? :)

Reply:
Removed Ukrainian. Will change list of languages in next PR.